_posts/dusan_andric/2018-05-25-icgc-jupyter.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Jupyter in ICGC"
```

Comment:
I would expand a bit on the title, for example: "Jupyter in ICGC: overview of an integration" or something similar